k9mail/src/androidTest/java/com/fsck/k9/helper/IdentityHelperTest.java

```diff
@@ -0,0 +1,141 @@
+package com.fsck.k9.helper;
+
+import android.content.Context;
+import android.test.ApplicationTestCase;
+import android.test.RenamingDelegatingContext;
+
+import com.fsck.k9.Account;
+import com.fsck.k9.Identity;
+import com.fsck.k9.K9;
+import com.fsck.k9.mail.Message;
+import com.fsck.k9.mail.Address;
+import com.fsck.k9.mail.internet.BinaryTempFileBody;
+import com.fsck.k9.mail.internet.MimeMessage;
+import com.fsck.k9.mailstore.ReconstructMessageFromDatabaseTest;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by yesalam on 7/3/17.
+ */
+
+public class IdentityHelperTest extends ApplicationTestCase<K9> {
```

Comment:
I don't see why this needs to be an instrumentation test. It should run fine using Robolectric on the JVM.